distributed/comm/core.py

```diff
@@ -221,8 +224,9 @@ def _raise(error):
         except EnvironmentError as e:
             error = str(e)
             if time() < deadline:
-                await asyncio.sleep(0.01)
-                logger.debug("sleeping on connect")
+                logger.debug("delaying before connect retry")
+                await asyncio.sleep(backoff)
+                backoff *= 1.5
```

Comment:
I would be tempted to change this to `2` for a more exponential backoff.

```suggestion
                backoff *= 2
```

Reply:
I suggest that we keep with 1.5.  We're not in a huge rush to grow the timeout.